### ISSUE TITLE
Include this repo as part of monthly meeting agenda?

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/monthly#7]
@ronanyeah @sohilpandya as we have now synchronised monthly dinners and business meetings, perhaps we could retire this repo and just raise issues in the repo for tracking business meetings (this is part of our current tidying-up exercise)?

**Comment by ronanyeah** [comment]
@sofer I'm fine with that as long as it is explicit that there is no requirement to attend the business meeting or even have any interest in it all. This is because my original goal for the dinner was to maintain and strengthen the alumni network, give people the opportunity to reconnect or touch base, or introduce potential new students to the community.

So it's more like "We usually have a dinner after the business meeting that is open to all FAC alumni, students and friends..." etc as opposed to the "FAC Business Meeting Dinner".

**Comment by sofer** [comment]
@ronanyeah I totally agree with that. Synchronising them just makes it easy to do both.
